src/js/components/Calendar/doc.js

```diff
@@ -100,6 +100,18 @@ to disable the previous and next buttons.
     locale: PropTypes.string
       .description('The locale to use.')
       .defaultValue('en-US'),
+    messages: PropTypes.shape({
+      previous: PropTypes.string,
+      next: PropTypes.string,
+    })
+      .description(
+        `Custom messages for Calendar. Used for accessibility by screen
+          readers.`,
+      )
+      .defaultValue({
+        previous: 'Moved to {date}',
+        next: 'Moved to {date}',
+      }),
```

Comment:
doc changes should be done on grommet-site since the Calendar doc has been updated to the new format. See https://github.com/grommet/grommet-site/pull/230